src/Sanity.Linq/BlockContent/SanityHtmlSerializers.cs

```diff
@@ -144,6 +144,8 @@ public Task<string> SerializeImageAsync(JToken input, SanityOptions options)
         {
             var asset = input["asset"];
             var imageRef = asset?["_ref"]?.ToString();
+            var assetValue = asset?["value"];
+            var imageAltText = assetValue?["altText"];
```

Comment:
Perhaps ` ?? ""` should be added to the end of this line to avoid imageAltText being null. Alternatively the alt tag should only be added if altImageText isn't null on the line further down.